batch/batch/driver/instance_pool.py

```diff
@@ -85,18 +87,19 @@ async def async_init(self):
         row = await self.db.select_and_fetchone('''
 SELECT worker_type, worker_cores, worker_disk_size_gb,
   worker_local_ssd_data_disk, worker_pd_ssd_data_disk_size_gb,
-  standing_worker_cores, max_instances, pool_size
+  standing_worker_cores, max_instances, pool_size, enable_standing_worker
 FROM globals;
 ''')
 
         self.worker_type = row['worker_type']
         self.worker_cores = row['worker_cores']
         self.worker_disk_size_gb = row['worker_disk_size_gb']
-        self.worker_local_ssd_data_disk = row['worker_local_ssd_data_disk']
+        self.worker_local_ssd_data_disk = bool(row['worker_local_ssd_data_disk'])
```

Comment:
Why do you need to add the `bool` here? Shouldn't the values coming from pymysql be booleans already?

Reply:
MySQL represents booleans as TINYINT(1), afaik. They come into Python as integers:
```python3
>>> x = l.run_until_complete(db.execute_and_fetchone('select TRUE as value'))
>>> x
{'value': 1}

```